### PR TITLE
rgw/admin: Add support for /info section

### PR DESCRIPTION
**Pull request by anoopcs9**
## Checklist
- [x] Added tests for features and functional changes
- [x] Public functions and types are documented
- [x] Standard formatting is applied to Go code
- [x] Is this a new API? Added a new file that begins with `//go:build ceph_preview`
- [x] Ran `make api-update` to record new APIs

based on https://github.com/ceph/go-ceph/pull/616

### CODE REVIEW

**Comment by phlogistonjohn** [comment]
You're probably already aware of this since you also restarted some of the builds but some of the jobs are failing with a somewhat mysterious error, possibly a failure to download go modules. I restarted it again but if it fails again I may just suggest waiting a day or more in case this is an issue on the go module proxy server side.

File: rgw/admin/info_test.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+func (suite *RadosGWTestSuite) TestGetInfo() {
+	suite.SetupConnection()
+	co, err := New(suite.endpoint, suite.accessKey, suite.secretKey, newDebugHTTPClient(http.DefaultClient))
+	assert.NoError(suite.T(), err)
+
+	suite.T().Run("test get rgw cluster/endpoint information", func(t *testing.T) {
+		info, err := co.GetInfo(context.Background())
+		assert.NoError(suite.T(), err)
+		assert.Equal(suite.T(), "rados", info.InfoSpec.StorageBackends[0].Name)
```
Comment (thotz) [inline rgw/admin/info_test.go:22]: Please add test case for cluster-id as well

Comment (anoopcs9) [reply rgw/admin/info_test.go:50]: Done.

**Review by thotz — DISMISSED** [review]

**Comment by anoopcs9** [comment]
> You're probably already aware of this since you also restarted some of the builds but some of the jobs are failing with a somewhat mysterious error, possibly a failure to download go modules. I restarted it again but if it fails again I may just suggest waiting a day or more in case this is an issue on the go module proxy server side.

May be `info=read` capability is not treated correctly before quincy.

**Review by thotz — APPROVED** [review]

**Review by phlogistonjohn — COMMENTED** [review]
Good find on the setup script issue. I am a bit confused by the addition of `launch_rgw` which is basically exactly the same as `launch_radosgw`. Why not either rename launch_radosgw to launch_rgw or keep launch_ragdosgw and have the new function named launch_radosgw2 ?

**Comment by phlogistonjohn** [comment]
Sorry for noticing things bit-by-bit, but I see that there's no Signed-off-by of yours @anoopcs9 .  The co-authored bits looks good but it only has Matt's name in the sign off line.

**Comment by phlogistonjohn** [comment]
^ That applies to the first 2 commits only

**Comment by phlogistonjohn** [comment]
To be clear: I would *add* one for your own. I would leave any existing sign offs

**Comment by anoopcs9** [comment]
> To be clear: I would _add_ one for your own. I would leave any existing sign offs

Updated individual commit messages with Sign Offs.

**Review by phlogistonjohn — APPROVED** [review]
While my RGW knowledge is shaky everything seems OK to me particularly WRT to i-dotting and t-crossing for go-ceph processes ;-)

**Review by thotz — APPROVED** [review]

**Pull request merged**